src/App.tsx

```diff
@@ -1,7 +1,10 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { PeoplePage } from './pages/PeoplePage/PeoplePage';
+import { Navbar } from './components/Navbar/Navbar';
 
 import './App.scss';
+import { HomePage } from './pages/HomePage/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage/NotFoundPage';
```

Comment:
There are tools for the automatic sorting of imports like extensions or default ones that can be included in your IDE. You can configure hotkeys, which not only sorts imports but also removes unused ones, and merges different lines with the same source into one line. Here is an example of what I'm using. The more helpful tools and hotkeys you'll learn the more productive and useful you can be for your future company.
![image](https://user-images.githubusercontent.com/45945732/233769132-537d468b-2dbe-4150-aa34-6f92ae9b2bc0.png)